application/dependencies.json

```diff
@@ -0,0 +1,180 @@
+{
+  "name": "bower-asset/inputmask",
+  "version": "3.3.11"
+}
+{
+  "name": "cebe/markdown",
+  "version": "1.2.1"
+}
+{
+  "name": "codemix/yii2-streamlog",
+  "version": "1.3.1"
+}
+{
+  "name": "doctrine/deprecations",
+  "version": "v1.1.1"
+}
+{
+  "name": "doctrine/lexer",
+  "version": "2.1.0"
+}
+{
+  "name": "egulias/email-validator",
+  "version": "3.2.6"
+}
+{
+  "name": "ezyang/htmlpurifier",
+  "version": "v4.16.0"
+}
+{
+  "name": "fillup/fake-bower-assets",
+  "version": "2.0.9"
+}
+{
+  "name": "firebase/php-jwt",
+  "version": "v6.8.0"
+}
+{
+  "name": "forevermatt/calc-api-sig",
```

Comment:
As a side note, I can't think of any reason we need this any more in this repo. We could check this codebase at some point to confirm that, and if not, then remove it from application/composer.json.